pennylane/plugins/tests/__init__.py

```diff
@@ -47,4 +32,163 @@
 For non-analytic tests, the tolerance of the assert statements
 is set to a high enough value to account for stochastic fluctuations. Flaky is used to automatically
 repeat failed tests.
+
+There are several methods for running the tests against a particular device (i.e., for
+``'default.qubit'``), detailed below.
+
+Using pytest
+------------
+
+.. code-block:: console
+
+    pytest path_to_pennylane_src/plugins/tests --device=default.qubit --shots=1234 --analytic=False
+
+The location of your PennyLane installation may differ depending on installation method and
+operating system. To find the location, you can use the :func:`~.get_device_tests` function:
+
+>>> from pennylane.plugins.tests import get_device_tests
+>>> get_device_tests()
+
+The pl-device-test CLI
+----------------------
+
+Alternatively, PennyLane provides a command line interface for invoking the device tests.
+
+.. code-block:: console
+
+    pl-device-test --device default.qubit --shots 1234 --analytic False
```

Comment:
Took me a while to figure out that one needs to `pip install -e .` to get this to work...

Strangely, 2 tests for default.qubit fail? Not essential here, but just wondering.

```
FAILED pennylane/plugins/tests/test_measurements.py::TestSample::test_sample_values_hermitian[device_kwargs0]
FAILED pennylane/plugins/tests/test_measurements.py::TestSample::test_sample_values_hermitian_multi_qubit[device_kwargs0]
```

Reply:
🤔

Reply:
> Took me a while to figure out that one needs to pip install -e . to get this to work...

Yep! The `setup.py` file needs to be run for the CLI script to be placed on the path.

> Strangely, 2 tests for default.qubit fail? Not essential here, but just wondering.

These two tests fail stochastically for me; looks like a tolerance issue. Increasing shots to 10000 fixes it :)